src/main/java/com/endava/internship/collections/StudentSet.java

```diff
@@ -1,84 +1,328 @@
 package com.endava.internship.collections;
 
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.Set;
+import java.util.*;
+import java.lang.reflect.Array;
 
 public class StudentSet implements Set<Student> {
+    private Entry<Student>[] buckets;
+    private int capacity, size;
+    private static final int INITIAL_CAPACITY = 16;
```

Comment:
static fields first

Reply:
and also add a new line after each statement

Reply:
Fixed